com.unity.render-pipelines.core/Tests/Editor/RemoveComponentTests.cs

```diff
@@ -0,0 +1,152 @@
+                [DisallowNull] Type componentToRemove,
+                [DisallowNull] Type[] componentsToAdd,
+                [DisallowNull] Type[] expectedComponents)
+                 => GenericRemoveComponent(gameObject, componentToRemove, componentsToAdd, expectedComponents, RemoveAdditionalComponent);
+        }
+
+        static TestCaseData[] s_RemoveComponentTestCaseDatas =
+        {
+            new TestCaseData(typeof(Banana), new Type[] {typeof(Banana), typeof(AdditionalBanana)}, Array.Empty<Type>())
+                .SetName("RemoveSingleAdditionalData"),
+            new TestCaseData(typeof(Banana), new Type[] {typeof(Banana), typeof(AdditionalBanana), typeof(AdditionalBananaColor)}, Array.Empty<Type>())
+                .SetName("RemovesMultipleAdditionalDatas"),
+            new TestCaseData(typeof(Apple), new Type[] {typeof(Banana), typeof(AdditionalBanana), typeof(Apple), typeof(AdditionalApple)}, new Type[] {typeof(Banana), typeof(AdditionalBanana)})
+                .SetName("OnlyRemoveItsAdditionalData")
+        };
+
+        [Test, TestCaseSource(nameof(s_RemoveComponentTestCaseDatas))]
+        public void RemoveComponent([DisallowNull] Type componentToRemove, [DisallowNull] Type[] componentsToAdd, [DisallowNull] Type[] expectedComponents)
+        {
+            Assert.True(Property.RemoveComponent(m_GameObject, componentToRemove, componentsToAdd, expectedComponents));
+        }
+
+        static TestCaseData[] s_RemoveAdditionalDataComponentTestCaseDatas =
+        {
+            new TestCaseData(typeof(AdditionalBanana), new Type[] {typeof(Banana), typeof(AdditionalBanana) }, Array.Empty<Type>())
+                .SetName("RemoveSingleAdditionalData"),
+            new TestCaseData(typeof(AdditionalBananaColor), new Type[] {typeof(Banana), typeof(AdditionalBanana), typeof(AdditionalBananaColor)}, Array.Empty<Type>())
+                .SetName("RemovesMultipleAdditionalDatas"),
+           new TestCaseData(typeof(AdditionalBananaColor), new Type[] {typeof(Banana), typeof(Banana), typeof(AdditionalBanana), typeof(AdditionalBananaColor)}, Array.Empty<Type>())
+                .SetName("RemovesMultipleComponent"),
+            new TestCaseData(typeof(AdditionalApple), new Type[] {typeof(Banana), typeof(AdditionalBanana), typeof(Apple), typeof(AdditionalApple)}, new Type[] {typeof(Banana), typeof(AdditionalBanana)})
+                .SetName("OnlyRemoveItsAdditionalData")
+        };
+
+        [Test, TestCaseSource(nameof(s_RemoveAdditionalDataComponentTestCaseDatas))]
+        public void RemoveAdditionalDataComponent([DisallowNull] Type componentToRemove, [DisallowNull] Type[] componentsToAdd, [DisallowNull] Type[] expectedComponents)
+        {
+            Assert.True(Property.RemoveAdditionalComponent(m_GameObject, componentToRemove, componentsToAdd, expectedComponents));
+        }
+
+        static TestCaseData[] s_TryGetComponentsToRemoveTestCaseDatas =
+         {
```

Comment:
Seems like there is an offset in this bracket. Check formating job to be sure

Reply:
It is green, maybe github is having something weird